gramex/apps/capture/chromecapture.js

```diff
@@ -129,7 +140,8 @@ async function render(q) {
     args.push('--proxy-server=' + proxy)
 
   if (typeof browser == 'undefined')
-    browser = await puppeteer.launch({args: args})
+    browser = await browser_setup(args)
```

Comment:
We need to make another check here. If `browser` is undefined, OR if `browser` is disconnected, we should run this setup.